policy/modules/system/mount.te

```diff
@@ -54,6 +54,7 @@ manage_dirs_pattern(mount_t, mount_runtime_t, mount_runtime_t)
 manage_files_pattern(mount_t, mount_runtime_t, mount_runtime_t)
 rw_files_pattern(mount_t, mount_runtime_t, mount_runtime_t)
 files_runtime_filetrans(mount_t, mount_runtime_t, dir, "mount")
+files_runtime_filetrans(mount_t, mount_runtime_t, file)
```

Comment:
I think we can merge this into a single line. The `"mount"` name is unnecessary.